bitnami/common/Chart.yaml

```
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
```

Not related but, could you add this?
```suggestion
  - https://github.com/bitnami/charts/tree/main/bitnami/common
  - https://github.com/bitnami/charts
```